plugins/basic/nexus-logging-plugin/src/main/java/org/sonatype/nexus/logging/rest/LoggersResource.java

```diff
@@ -76,6 +77,13 @@ public LoggersResource(final LoggingConfigurator configurator) {
   }
 
   /**
+   * Helper to check precondition of logger name.
+   */
+  private static void checkLoggerName(final String name) {
+    checkArgument(name.equals(StringEscapeUtils.escapeHtml(name)), "Invalid logger name");
```

Comment:
this could technically be the same regex used in the UI, but this should also do the job.